kubernetes/base/statefulset.yaml

```
@@ -17,6 +17,9 @@ spec:
     spec:
       serviceAccountName: chat-service
       volumes:
+      - name: statsd-mapping
```

```suggestion
      # Configuration files for the statsd_exporter sidecar
      - name: statsd-mapping
```
Although, where does this configmap come from ?

---

I forgot to push the changes, pushing them now